src/components/Editor/Runners/PythonRunner/OutputViewToggle.js

```diff
@@ -25,32 +30,30 @@ const OutputViewToggle = () => {
   };
 
   return (
-    <div
-      className={`output-view-toggle`}
-      disabled={codeRunTriggered || drawTriggered}
-    >
-      <Button
-        className={`btn--small output-view-toggle__button output-view-toggle__button--tabbed${
-          isSplitView ? "" : " output-view-toggle__button--active"
-        }`}
-        buttonOuter
-        disabled={codeRunTriggered || drawTriggered}
-        label={t("outputViewToggle.buttonTabLabel")}
-        title={t("outputViewToggle.buttonTabTitle")}
-        ButtonIcon={TabbedViewIcon}
-        onClickHandler={switchToTabbedView}
-      />
-      <Button
-        className={`btn--small output-view-toggle__button output-view-toggle__button--split${
-          isSplitView ? " output-view-toggle__button--active" : ""
-        }`}
-        buttonOuter
-        disabled={codeRunTriggered || drawTriggered}
-        label={t("outputViewToggle.buttonSplitLabel")}
-        title={t("outputViewToggle.buttonSplitTitle")}
-        ButtonIcon={SplitViewIcon}
-        onClickHandler={switchToSplitView}
-      />
+    <div className="output-view-toggle">
+      {isSplitView ? (
+        <Button
+          className={"btn--tertiary output-view-toggle__button"}
+          buttonText={isMobile ? null : t("outputViewToggle.buttonTabLabel")}
+          disabled={codeRunTriggered || drawTriggered}
+          label={t("outputViewToggle.buttonTabLabel")}
+          title={t("outputViewToggle.buttonTabLabel")}
+          ButtonIcon={TabbedViewIcon}
+          buttonIconPosition="right"
+          onClickHandler={switchToTabbedView}
+        />
+      ) : (
+        <Button
+          className={"btn--tertiary output-view-toggle__button"}
+          buttonText={isMobile ? null : t("outputViewToggle.buttonSplitLabel")}
+          disabled={codeRunTriggered || drawTriggered}
+          label={t("outputViewToggle.buttonSplitLabel")}
+          title={t("outputViewToggle.buttonSplitTitle")}
+          ButtonIcon={SplitViewIcon}
+          buttonIconPosition="right"
+          onClickHandler={switchToSplitView}
+        />
+      )}
```

Comment:
🤔 this is all quite chunky and mostly the same, just wondering if adding some of the logic in the attributes might be better or easier to read? I'm easy either way though, just thinking out loud.

```
      <Button
        className={"btn--tertiary output-view-toggle__button"}
        disabled={codeRunTriggered || drawTriggered}
        label={t("outputViewToggle.buttonTabLabel")}
        title={t("outputViewToggle.buttonTabLabel")}
        ButtonIcon={isSplitView ? TabbedViewIcon : SplitViewIcon}
        buttonIconPosition="right"
        onClickHandler={isSplitView ? switchToTabbedView : switchToSplitView}
        {...(!isMobile && {
          buttonText: isSplitView
            ? t("outputViewToggle.buttonTabLabel")
            : t("outputViewToggle.buttonSplitLabel"),
        })}
      />
```

Reply:
Ahhh yes I didn't think of this, but that should tidy things up quite a bit, thanks!